tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"include": ["./src/**/*"],
+	"exclude": ["node_modules"],
+	"compilerOptions": {
+		"target": "es2016",
+		"module": "ES2015" /* Specify what module code is generated. */,
+		"rootDir": "./src" /* Specify a file that bundles all outputs into one JavaScript file. If 'declaration' is true, also designates a file that bundles all .d.ts output. */,
```

Comment:
Adding `"moduleResolution": "node",` to compilerOptions will sort the VS Code complaint about the dotenv import.